environment.yml

```diff
@@ -22,7 +22,7 @@ dependencies:
   - ecmwf-api-client
   - eofs
   - esmpy >=8.6.0  # github.com/SciTools-incubator/iris-esmf-regrid/pull/342
```

Comment:
@ehogan this is incompatible with the pin in ESMValCore 2.11.0rc2 (there it's <8.6.0) - we either change it here, or there before the main release, but you've done all the tests with <8.6.0, so I'd keep it like that in Core, and just free this one here :+1:

Reply:
It's only like this in the `main` branch V, not in the release branch: https://github.com/ESMValGroup/ESMValTool/blob/v2.11.x/environment.yml#L24.

Reply:
ah excellent then! Though we ought do something about it in the `main` branch too, since the user's env don't solve this way